mitmproxy/protocol/http2.py

```diff
@@ -96,15 +96,16 @@ def __init__(self, ctx, mode):
         self.server_to_client_stream_ids = dict([(0, 0)])
         self.client_conn.h2 = SafeH2Connection(self.client_conn, client_side=False, header_encoding=False)
 
-        # make sure that we only pass actual SSL.Connection objects in here,
-        # because otherwise ssl_read_select fails!
-        self.active_conns = [self.client_conn.connection]
-
```

Comment:
Ehm - there shouldn't be any real changes, just cosmetics to make use of the same "layout" as WebSockets and RawTCP. That's why I included it in this PR, because it is only style-related to make it look the same as websockets.